### DISCUSSION TITLE
插件简洁有力！让我燃起打印成册的想法，能否开发合并打印功能？

### DISCUSSION THREAD

**Discussion by limitoec** [l1xnan/obsidian-better-export-pdf#166] [Ideas]
Twitter无意看到，配合blue Topaz主题，感觉太好看了。打印成为了乐趣。
然而希望燃气，我希望把一个folder内部的都打印出来。能否开发合并打印的功能？或者新开发一个也可以。

**Comment by l1xnan** [comment]
感谢支持，目前支持将整个目录导出为一个PDF，见：[多文件导出](https://github.com/l1xnan/obsidian-better-export-pdf/blob/master/README.zh.md#%E5%A4%9A%E6%96%87%E4%BB%B6%E5%AF%BC%E5%87%BA)，可以先试用一下，就是打印成册可能还有其他细节上的要求，我们可以再探讨

**Reply by limitoec** [reply]
断断续续折腾了几天，还是不行。这个toc好像不能解决我的问题。我的最终想法是希望better pdf这个 把我生成的所有文章的highlights做成一个pdf打印成pdf，然后用打印机打印出来经常翻翻。
用 【整个目录导出为pdf】 用起来好像不行，一直在folder内的笔记之间跳动。就差最后一步了。
希望以后能把这个做起来。
目前我[ Advanced Merger ] 勉强能把folder内的笔记（主要都是highlight摘要 短小精悍）合并成一个md文件。然后用你的插件来打印。
目前还不完美。因为那个插件把 front matter 的内容都弄上去了，什么id之类的，还是看起来不完美。
只能靠你了。你的这个插件第一次用就看上去很好看。我希望用你的插件把我的一些笔记都 合并打印了 成一个pdf，然后淘宝打印出来，自己慢慢翻看。也对眼睛好。
谢谢！

**Reply by l1xnan** [reply]
highlights 指的是什么? 跳动是因为插件在逐个渲染笔记，有点慢，一个笔记快的的好几百ms，慢地要几秒，应该优化个进度条

**Reply by limitoec** [reply]
highlights 就是我把网文用一些插件比如omnivore做的摘要。这种文章很多。原文就算了。但是我的摘要，我想全部用你的better-export-pdf打印出来。因为你的插件打出来的确很赏心悦目！！！！！非常好看！我就想把这些highlights合并打印，不要天天用电脑看了。
![image](https://github.com/l1xnan/obsidian-better-export-pdf/assets/105053681/889ea85f-bd6a-4a56-b801-3e2b28974922)

**Reply by l1xnan** [reply]
这种应该支持的，就是你笔记比较多，耗时应该比较久，可以先用几个笔记测试下，速度上后面再想办法优化下

**Reply by limitoec** [reply]
果然，我之前是耐心不够。。还真是可以。一个folder下面大概有110个笔记。耐心等了几分钟。还真可以了。
![image](https://github.com/l1xnan/obsidian-better-export-pdf/assets/105053681/ea50be62-964b-4b42-b06c-4ce60392ea62)
感觉不错。非常感谢！
我可以打印去了。

**Reply by limitoec** [reply]
同样的工作量,在mac上 M2 Pro处理器居然只需要 不到一分钟.........Intel体验堪忧.
